examples/tensorflow/segmentation/evaluation.py

```diff
@@ -35,7 +35,8 @@
 
 
 def get_argument_parser():
-    parser = get_common_argument_parser(weights=False,
+    parser = get_common_argument_parser(without_mode=True,
```

Comment:
```suggestion
    parser = get_common_argument_parser(mode=False,
```

Reply:
Ok, but it really is all the same in Python, especially for the things that most likely will only be changed by us, internally.